src/main/java/pl/edu/zut/mad/schedule/LessonsAdapter.kt

```diff
@@ -0,0 +1,65 @@
+package pl.edu.zut.mad.schedule
+
+import android.content.Context
+import android.support.v4.content.ContextCompat
+import android.view.View
+import android.widget.TextView
+import com.ognev.kotlin.agendacalendarview.models.CalendarEvent
+import com.ognev.kotlin.agendacalendarview.render.DefaultEventAdapter
+import pl.edu.zut.mad.schedule.data.model.ui.Lesson
+import pl.edu.zut.mad.schedule.data.model.ui.LessonEvent
+import java.text.SimpleDateFormat
+import java.util.*
```

Comment:
I prefer to use explicit imports. You could change it in your settings. It is up to you.

[Here](https://github.com/shyiko/ktlint) is nice tool, ktlint where this is one of the rules. Try it if you like